utils/constants.jsx

```diff
@@ -461,12 +463,13 @@ export const SocketEvents = {
 };
 
 export const TutorialSteps = {
-    POST_POPOVER: 0,
```

Comment:
A longstanding issue with these tutorial steps is that the backend storage for them is a single int stored in a single preference This means our concept of "tip already seen" is limited to whatever the current number is. This makes things like reordering steps problematic, because users in middle of the tutorial will miss some steps they haven't seen, and others will see steps they already say. And in this case, it will cause people who already saw the Settings tip to see it again because clicking the "next/got it/ok" button increments the step by one. 

When I brought this up a couple months ago, @AGMETEOR had a good idea of changing the storage to be more similar to the various `recommended_next_steps` preferences that store a single preference per completed item. E.g. the values that get sent to the client from the DB look like this:
```json
[
  {"user_id":"nz3xk4hq9ird78nxapab3wt6io","category":"recommended_next_steps","name":"complete_profile","value":"true"},
  {"user_id":"nz3xk4hq9ird78nxapab3wt6io","category":"recommended_next_steps","name":"hide","value":"true"},
  {"user_id":"nz3xk4hq9ird78nxapab3wt6io","category":"recommended_next_steps","name":"invite_members","value":"true"},
  {"user_id":"nz3xk4hq9ird78nxapab3wt6io","category":"recommended_next_steps","name":"notification_setup","value":"true"},
]
``` 

Long term, I think we should start storing the tips like that, and have a migration that runs that converts to the "store a single preference for each tip" system.

For the sake of finishing this PR & A/B test, I think we can get by with making the value of `ADD_FIRST_CHANNEL` be `-1.

Reply:
I agree that for a permanent solution storing the `recomended_next_step` as string name value could potentially work better, but that is not a simple change since it implies modifying all the function related that use the number increase as a way of tracking what tip goes next. On the other hand, changing it to -1 value, implies to modify also the starting screen, initial step, current step and several other values that for the sake of an A/B test I consider more risky and error prone than only modify the steps so the `create_first_channel` is the 0 value.

Reply:
I really think we need to make `ADD_FIRST_CHANNEL` have a value of `-1`. Users who finish the tour points by going through them have a value of `6`. Only users who finish by skipping by opting out have a value of `FINISHED`/`999.` Shifting all the `TutorialSteps` values means everyone gets shifted back one tutorial step. So for every user, whether they are an admin or not, if they have finished the tutorials normally they will re-see the settings tip.

Reply:
+1 on making sure we don't interfere with existing users and have them all be getting the final `Settings` tip again due to the number change.